### PR TITLE
git, git-rebase: add Chinese translation

### PR DESCRIPTION
**Pull request by kasodesyn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
Welcome to tldr @kasodesyn, Thanks for your contribution. 
I will be adding reviews to fix the linter errors. You can later batch all the suggestions and make them as a single commit. 
Take this as a reference, Your file without any [errors](https://github.com/kbdharun/tldr/commit/5b35f95bf0f1048c04fee0d4b9af67606c6ff8df),  and with a successful run of [CI](https://github.com/kbdharun/tldr/actions/runs/3071396271/jobs/4962022881).

File: pages.zh/common/git-rebase.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# git rebase
+
+> 将 commits 从一个分支合并到另一个分支上。
```
Comment (kbdharun) [inline pages.zh/common/git-rebase.md:3]:
```suggestion
> 将 commits 从一个分支合并到另一个分支上。.
```

File: pages.zh/common/git-rebase.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。
```
Comment (kbdharun) [inline pages.zh/common/git-rebase.md:4]:
```suggestion
> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。.
```

File: pages.zh/common/git-rebase.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+> 更多信息: <https://git-scm.com/docs/git-rebase>
```
Comment (kbdharun) [inline pages.zh/common/git-rebase.md:5]:
```suggestion
> 更多信息: <https://git-scm.com/docs/git-rebase>.
```

File: pages.zh/common/git.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统。
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。
+> 更多信息：<https://git-scm.com/>.
+
+- 检查 git 的版本号：
+
```
Comment (kbdharun) [inline pages.zh/common/git.md:8]:
```suggestion

```

File: pages.zh/common/git.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- 检查 git 的版本号：
+ 
+`git --version`
+
+- 显示帮助文档：
+
+`git --help`
+
+- 显示 git 子命令的详细帮助文档（如 clone, add, push, log 等子命令）：
+
```
Comment (kbdharun) [inline pages.zh/common/git.md:16]:
```suggestion

```

File: pages.zh/common/git.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统。
```
Comment (kbdharun) [inline pages.zh/common/git.md:3]:
```suggestion
> 分布式版本控制系统。.
```

File: pages.zh/common/git.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。
```
Comment (kbdharun) [inline pages.zh/common/git.md:4]:
```suggestion
> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。.
```

File: pages.zh/linux/git-rebase.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
+# git rebase
+
+> 将 commits 从一个分支合并到另一个分支上。
```
Comment (kbdharun) [inline pages.zh/linux/git-rebase.md:3]:
```suggestion
> 将 commits 从一个分支合并到另一个分支上。.
```

File: pages.zh/linux/git-rebase.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。
```
Comment (kbdharun) [inline pages.zh/linux/git-rebase.md:4]:
```suggestion
> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。.
```

File: pages.zh/linux/git-rebase.md [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+> 更多信息: <https://git-scm.com/docs/git-rebase>
```
Comment (kbdharun) [inline pages.zh/linux/git-rebase.md:5]:
```suggestion
> 更多信息: <https://git-scm.com/docs/git-rebase>.
```

File: pages.zh/linux/git.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统。
```
Comment (kbdharun) [inline pages.zh/linux/git.md:3]:
```suggestion
> 分布式版本控制系统。.
```

File: pages.zh/linux/git.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。
```
Comment (kbdharun) [inline pages.zh/linux/git.md:4]:
```suggestion
> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。.
```

File: pages.zh/linux/git.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+> 更多信息：<https://git-scm.com/>.
+
+- 检查 git 的版本号：
+
```
Comment (kbdharun) [inline pages.zh/linux/git.md:8]:
```suggestion

```

File: pages.zh/linux/git.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+- 检查 git 的版本号：
+ 
+`git --version`
+
+- 显示帮助文档：
+
+`git --help`
+
+- 显示 git 子命令的详细帮助文档（如 clone, add, push, log 等子命令）：
+
```
Comment (kbdharun) [inline pages.zh/linux/git.md:16]:
```suggestion

```

File: pages.zh/osx/git-rebase.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
+# git rebase
+
+> 将 commits 从一个分支合并到另一个分支上。
```
Comment (kbdharun) [inline pages.zh/osx/git-rebase.md:3]:
```suggestion
> 将 commits 从一个分支合并到另一个分支上。.
```

File: pages.zh/osx/git-rebase.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。
```
Comment (kbdharun) [inline pages.zh/osx/git-rebase.md:4]:
```suggestion
> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。.
```

File: pages.zh/osx/git-rebase.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+> 更多信息: <https://git-scm.com/docs/git-rebase>
```
Comment (kbdharun) [inline pages.zh/osx/git-rebase.md:5]:
```suggestion
> 更多信息: <https://git-scm.com/docs/git-rebase>.
```

File: pages.zh/osx/git.md [hunk 18]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统。
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。
+> 更多信息：<https://git-scm.com/>.
+
+- 检查 git 的版本号：
+
```
Comment (kbdharun) [inline pages.zh/osx/git.md:8]:
```suggestion

```

File: pages.zh/osx/git.md [hunk 19]
```diff
@@ -0,0 +1,29 @@
...
+- 检查 git 的版本号：
+ 
+`git --version`
+
+- 显示帮助文档：
+
+`git --help`
+
+- 显示 git 子命令的详细帮助文档（如 clone, add, push, log 等子命令）：
+
```
Comment (kbdharun) [inline pages.zh/osx/git.md:16]:
```suggestion

```

File: pages.zh/osx/git.md [hunk 20]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统。
```
Comment (kbdharun) [inline pages.zh/osx/git.md:3]:
```suggestion
> 分布式版本控制系统。.
```

File: pages.zh/osx/git.md [hunk 21]
```diff
@@ -0,0 +1,29 @@
...
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。
```
Comment (kbdharun) [inline pages.zh/osx/git.md:4]:
```suggestion
> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。.
```

File: pages.zh/windows/git-rebase.md [hunk 22]
```diff
@@ -0,0 +1,37 @@
+# git rebase
+
+> 将 commits 从一个分支合并到另一个分支上。
```
Comment (kbdharun) [inline pages.zh/windows/git-rebase.md:3]:
```suggestion
> 将 commits 从一个分支合并到另一个分支上。.
```

File: pages.zh/windows/git-rebase.md [hunk 23]
```diff
@@ -0,0 +1,37 @@
...
+> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。
```
Comment (kbdharun) [inline pages.zh/windows/git-rebase.md:4]:
```suggestion
> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 commit，表示合并完成。.
```

File: pages.zh/windows/git-rebase.md [hunk 24]
```diff
@@ -0,0 +1,37 @@
...
+> 更多信息: <https://git-scm.com/docs/git-rebase>
```
Comment (kbdharun) [inline pages.zh/windows/git-rebase.md:5]:
```suggestion
> 更多信息: <https://git-scm.com/docs/git-rebase>.
```

File: pages.zh/windows/git.md [hunk 25]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统。
```
Comment (kbdharun) [inline pages.zh/windows/git.md:3]:
```suggestion
> 分布式版本控制系统。.
```

File: pages.zh/windows/git.md [hunk 26]
```diff
@@ -0,0 +1,29 @@
...
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。
```
Comment (kbdharun) [inline pages.zh/windows/git.md:4]:
```suggestion
> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅。.
```

File: pages.zh/windows/git.md [hunk 27]
```diff
@@ -0,0 +1,29 @@
...
+> 更多信息：<https://git-scm.com/>.
+
+- 检查 git 的版本号：
+
```
Comment (kbdharun) [inline pages.zh/windows/git.md:8]:
```suggestion

```

File: pages.zh/windows/git.md [hunk 28]
```diff
@@ -0,0 +1,29 @@
...
+- 检查 git 的版本号：
+ 
+`git --version`
+
+- 显示帮助文档：
+
+`git --help`
+
+- 显示 git 子命令的详细帮助文档（如 clone, add, push, log 等子命令）：
+
```
Comment (kbdharun) [inline pages.zh/windows/git.md:16]:
```suggestion

```

**Review by kbdharun — CHANGES_REQUESTED** [review]

**Comment by kbdharun** [comment]
> Welcome to tldr @kasodesyn, Thanks for your contribution. I will be adding reviews to fix the linter errors. You can later batch all the suggestions and make them as a single commit. Take this as a reference, Your file without any [errors](https://github.com/kbdharun/tldr/commit/5b35f95bf0f1048c04fee0d4b9af67606c6ff8df), and with a successful run of [CI](https://github.com/kbdharun/tldr/actions/runs/3071396271/jobs/4962022881).

I have added the reviews for fixing linter errors, you can batch all the suggestions and make it as a single commit to avoid multiple CI/workflow runs.

**Comment by kasodesyn** [comment]
@kbdharun Thank you for your careful inspection, I have realized the differences in symbols between countries and have made changes to pass the CI, this is my first submission and am very excited, thank you very much!

**Comment by kbdharun** [comment]
Hey, Can you tell me what editor you used to create this file and has the line endings been set to Unix?

**Comment by kbdharun** [comment]
Yay, Now that all linter errors has been fixed let me review the translation.

**Comment by kasodesyn** [comment]
@kasodesyn I have pass all the test!ohhhhhh~~~~

**Comment by kbdharun** [comment]
Hey, I noticed that the same pages are present in files on each platform. 
We currently have it in common only because these commands work on all platforms and don't require separate pages in each platform. 
You could remove the pages from other platforms keeping pages in `common` alone.

File: pages.zh/common/git.md [hunk 29]
```diff
@@ -0,0 +1,29 @@
...
+
+`git help {{子命令}}`
+
+- 执行 git 的子命令:
+
+`git {{子命令}}`
+
+- 在自定义的 git 仓库根路径下执行子命令:
+
+`git -C path/to/repo {{子命令}}`
```
[the remark this answers is not in the document]
Comment (kasodesyn) [reply pages.zh/common/git.md]: Thanks for the suggestion, I have already added.

File: pages.zh/common/git.md [hunk 30]
```diff
@@ -0,0 +1,29 @@
...
+
+- 在给定参数条件下，执行 git 的子命令:
+
+`git -c 'config.key=value' {{子命令}}`
```
[the remark this answers is not in the document]
Comment (kasodesyn) [reply pages.zh/common/git.md]: Thanks for the suggestion, I have already added.

File: pages.zh/common/git-rebase.md [hunk 31]
```diff
@@ -0,0 +1,37 @@
...
+
+`git rebase --skip`
+
+- 终止正在执行中的合并任务（例如：对于正处于解决冲突中的任务，将其打断，恢复到合并前的状态）:
+
+`git rebase --abort`
+
+- 将分支的部分 commits 生成新的 commit，移动到新分支的头节点:
+
+`git rebase --onto {{目标分支}} {{当前分支}}`
```
[the remark this answers is not in the document]
Comment (kasodesyn) [reply pages.zh/common/git-rebase.md:29]: The statement of `{{new_base}} {{old_base}}` is misunderstood, and `{target branch}} {{current branch}}` in Chinese is better understood.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks

**Comment by kasodesyn** [comment]
Thank you for your valuable comments, I have re-reviewed and submitted the latest document, thanks.

File: pages.zh/common/git.md [hunk 32]
```diff
@@ -0,0 +1,29 @@
...
+
+- 检查 git 的版本号:
+
+`git --version`
+
+- 显示帮助文档：
+
+`git --help`
+
+- 显示 git 子命令的详细帮助文档（如 clone, add, push, log 等子命令）:
```
Comment (kbdharun) [inline pages.zh/common/git.md:15]:
```suggestion
- 显示 git 子命令的详细帮助文档（如 `clone`, `add`, `push`, `log` 等子命令）:
```

**Comment by kasodesyn** [comment]
Thank you, this is the first time I submit a pull request, I will check more carefully in the later.

File: pages.zh/common/git.md [hunk 33]
```diff
@@ -0,0 +1,29 @@
+# git
+
+> 分布式版本控制系统.
+> 类似如 commit、add、branch、checkout、push 等子命令都有自己的使用文档，可以通过 tldr git {{子命令}} 的形式查阅.
```
Comment (kbdharun) [inline pages.zh/common/git.md:4]:
```suggestion
> 类似如 `commit`、`add`、`branch`、`checkout`、`push` 等子命令都有自己的使用文档，可以通过 `tldr git {{子命令}}` 的形式查阅.
```

**Review by kbdharun — APPROVED** [review]
Just a small suggestion

File: pages.zh/common/git.md (hunk 33 above)
Comment (kasodesyn) [reply pages.zh/common/git.md]: Thanks for the suggestion, I have already resolved.

**Comment by kasodesyn** [comment]
Thanks for the suggestion, I have already resolved.

**Comment by blueskyson** [comment]
@kbdharun `。` is Chinese period, so the original period usage is right. We only use Latin period `.` in more information links. For detailed rules please see [Chinese-Specific Rules](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#chinese-specific-rules).

File: pages.zh/common/git-rebase.md [hunk 34]
```diff
@@ -0,0 +1,37 @@
+# git rebase
+
+> 将 commits 从一个分支合并到另一个分支上.
```
Comment (blueskyson) [inline pages.zh/common/git-rebase.md:3]: Use Chinese periods in Chinese sentences except "more information: <...>" line.
```suggestion
> 将 commits 从一个分支合并到另一个分支上。
```

File: pages.zh/common/git-rebase.md [hunk 35]
```diff
@@ -0,0 +1,37 @@
...
+> 常用于跨分支的 commits 合并，在被合并分支的最头部构建新的 `commit`，表示合并完成.
+> 更多信息: <https://git-scm.com/docs/git-rebase>.
+
+- 在另一个分支的头节点合并当前分支:
+
+`git rebase {{目标分支}}`
+
+- 启动交互式的合并任务，允许对提交的内容进行重新排序、省略、合并或修改:
+
+`git rebase -i {{目标分支或commit的hash}}`
```
Comment (blueskyson) [inline pages.zh/common/git-rebase.md:13]: Place spaces between Chinese and Latin characters.
```suggestion
`git rebase -i {{目标分支或 commit 的 hash}}`
```

**Review by blueskyson — COMMENTED** [review]
Sorry to ask you turn the `.` back to `。` except "more information: <...>" lines. See [style-guide.zh.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.zh.md) for Chinese rules.

**Comment by kasodesyn** [comment]
好的，在吃饭，回去就改，辛苦二位。

**Comment by kasodesyn** [comment]
@blueskyson I have resubmitted for hard review. Thanks to @kbdharun , with his help, for letting me know the submission specifications and some missing parts, again!

**Review by kbdharun — APPROVED** [review]

**Comment by kasodesyn** [comment]
@blueskyson 辛苦处理哦～

**Review by blueskyson — APPROVED** [review]

**Pull request merged**